connect/src/com/telenor/connect/utils/TurnOnMobileDataAnalytics.java

```diff
@@ -7,14 +7,14 @@
 
 import java.io.Serializable;
 
-public class EnableMobileDataDialogAnalytics implements Serializable {
+public class TurnOnMobileDataAnalytics implements Serializable {
```

Comment:
Maybe it should be "TurnOnMobileData**Dialog**Analytics"?

Reply:
It was intentional, but I think perhaps I've changed my mind and agree with you. The dialog is core to the feature.